### PR TITLE
End point to delete a todo added

### PR DESCRIPTION
**Pull request by vinay-jtc**
### Endpoint to delete a todo added.
URI: DELETE /todos/:id
Response Status: 204 if todo item exist and id is valid mongo id

## Database schema changes

- N/A

### Automated test cases added

- Test status code 204 if todo item exist and id is valid mongo id
- Test status code 400 if id isn't valid mongo id
- Test status code 404 if todo item does not exist and id is valid mongo id

### Manual test cases run

-  Run by postman

### CODE REVIEW

**Comment by coder1033** [comment]
We don't delete any data from database.

Add a boolean `isActive` in `todoItemSchema`, which is by default `true`.
If `isActive` is `true`, then data is available for client.
If `isActive` is `false`, then data is not available for client.

**Comment by vinay156** [comment]
I understand this and I will consider this from the upcoming project.

**Comment by vinay-jtc** [comment]
I understand this and i will consider it in upcominng project.

File: test/spec/todo/todo.spec.ts [hunk 1]
```diff
@@ -45,22 +47,28 @@ describe('POST /todos', () => {
...
 
-describe('GET /todos', () => {
-  it('we should fatch all the todo items', async () => {
-    const res = await chai.request(expressApp).get('/todos');
 
-    expect(res).to.have.status(200);
-    expect(res.body).to.be.an('array');
+describe("DELETE /todos/:id", () => {
+  it("should return 204 if todo exists and id is valid mongo id", async () => {
+    let todo = await testAppContext.TodoItemRepository.save(
```
Comment (coder1033) [inline test/spec/todo/todo.spec.ts]: `const`

**Review by coder1033 — APPROVED** [review]

**Comment by coder1033** [comment]
What is `done` in your Manual testing description?

File: test/spec/todo/todo.spec.ts (hunk 1 above)
Comment (vinay-jtc) [reply test/spec/todo/todo.spec.ts]: Updated

**Pull request merged**